src/server/index.js

```diff
@@ -15,8 +15,8 @@ app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
 // Add your routes here
-
-
+const pokemonRouter = require("./routers/pokemon")
+app.use("/pokemon", pokemonRouter)
```

Comment:
this route is used to populate the pokemon database since we cannot fetch inside the seed. We will need to change it so that users cannot call the initial population from their URL!

Reply:
makes sense, hopefully we find a way!

Reply:
Good observation @Can-Tina! Definitely remember to take this route into account down the line.

I want to remind everyone that this is almost certainly not the way to solve this problem you had in the real world, most likely you'd write some kind of setup script in bash to take care of this.